eclipse-scout-core/src/modeselector/ModeSelector.ts

```diff
@@ -57,6 +73,38 @@ export class ModeSelector<TModeRef = any> extends Widget implements ModeSelector
     this._renderModes();
   }
 
+  protected override _renderEnabled() {
+    super._renderEnabled();
+    this._provideTabIndex();
+  }
+
+  protected _provideTabIndex() {
```

Comment:
Fixed by overriding _renderTabbable in Mode.ts and not checking for enabled there

Reply:
As you mentioned, this method was not working correctly because the child actions enabled state was not computed yet. It worked for radio group because the method was called twice, which is not done for modeselector. I moved calling of the method from (i think it was) selectEnabled to recomputeEnabled. At this place, child action enabled state is already computed